agent/http.go

```diff
@@ -194,6 +201,12 @@ func (s *HTTPServer) wrap(handler func(resp http.ResponseWriter, req *http.Reque
 			fmt.Fprint(resp, errMsg)
 		}
 
+		if s.blacklist.IsDisallowed(req.URL.Path) {
+			err := fmt.Errorf("Permission denied, endpoint is disabled")
```

Comment:
We should probably type these errors instead of depending on magic words but that is probably a larger change.